.github/workflows/run-gradle.yml

```diff
@@ -38,22 +47,25 @@ jobs:
                distribution: "temurin"
                java-version: "17"
 
-         -  uses: gradle/actions/setup-gradle@v3
+         -  name: Setup Gradle
+            uses: gradle/actions/setup-gradle@v3
+            with:
+               gradle-home-cache-cleanup: true
 
          -  name: Run tests
             run: ./gradlew ${{ inputs.task }} --scan
             shell: bash
 
-         - name: Upload build reports
-           if: failure()
-           uses: actions/upload-artifact@v4
-           with:
-              name: build-reports-${{ runner.os }}-${{ github.action }}-${{ github.run_id }}
-              path: |
-                 **/build/reports/
-                 **/*.hprof
-                 **/*.log
-              if-no-files-found: ignore
+         -  name: Upload build reports
+            if: failure()
+            uses: actions/upload-artifact@v4
+            with:
+               name: build-reports-${{ runner.os }}-${{ github.action }}-${{ github.run_id }}
+               path: |
+                  **/build/reports/
+                  **/*.hprof
+                  **/*.log
+               if-no-files-found: ignore
```

Comment:
```suggestion
         - name: Upload build reports
           if: failure()
           uses: actions/upload-artifact@v4
           with:
              name: build-reports-${{ runner.os }}-${{ github.action }}-${{ github.run_id }}
              path: |
                 **/build/reports/
                 **/*.hprof
                 **/*.log
              if-no-files-found: ignore
```

formatting nitpick

Reply:
Yeah, normally I'd do as suggested. But I actually prefer consistency and auto-formatting over slight personal preferences, and that change was what IJ wanted (me) to do. It is consistent with the other sections in that file.

Maybe we could change `.editorconfig` to accomodate a different style and re-format all `yaml`? Until then, I'd leave it at that for now.